### ISSUE TITLE
Not be able to change username

### ISSUE THREAD

**Issue by chinglinwen** [kennygrant/gohackernews#16]
Expect a place to change username, Currently I have not found the place to do that

**Comment by kennygrant** [comment]
Yes, fair point, it'd be good to be able to edit your own username. Admins are able to for their user but this is not enabled for end users.

**Comment by kennygrant** [comment]
Note to self: Usernames should be a public identifier, which the user can change whenever they like and we should allow collisions (since many people have the same name) and have some way of differentiating them by id or icon (for example little autogenerated icons). 

Also fix the horrible storing of usernames on posted comments and do that properly with joins.
